odl/solvers/nonsmooth/primal_dual_hybrid_gradient.py

```diff
@@ -22,14 +22,15 @@
 from odl.operator import Operator
 
 
-__all__ = ('chambolle_pock_solver',)
+__all__ = ('pdhg',)
 
 
 # TODO: add dual gap as convergence measure
 # TODO: diagonal preconditioning
 
-def chambolle_pock_solver(x, f, g, L, tau, sigma, niter, **kwargs):
-    """Chambolle-Pock algorithm for non-smooth convex optimization problems.
+def pdhg(x, f, g, L, tau, sigma, niter, **kwargs):
+    """Primal-dual hybrid gradient algorithm for non-smooth convex optimization
+    problems.
```

Comment:
This should fit on one line (perhaps simply remove "problems")

Reply:
Done